src/components/swipe/swipe.js

```diff
@@ -62,9 +80,15 @@
  * The md-swipe-down directive allows you to specify custom behavior when an element is swiped
  * down.
  *
+ * > You can see this in action on the <a ng-href="demo/swipe">demo page</a> (Look at the Developer Tools console while swiping).
```

Comment:
Please make this consistent with the others, I.e. this message under the Notes.

Reply:
My bad. Fixed this.
Thanks a lot - This is my first contribution to a big/popular open source repo, so very excited about that! 🎉